perses/tests/test_relative.py

```diff
@@ -1157,5 +1210,8 @@ def test_unsampled_endstate_energies_GPU():
     # Alanine dipeptide in solvent -- Run MD energy validation test
     run_unsampled_endstate_energies('ala-dipeptide', use_point_energies=False, use_md_energies=True)
 
-    # Apo barstar -- Run point and MD energy validation test
+    # Apo barstar -- Run point and MD energy validation tests
     run_unsampled_endstate_energies('barstar', use_point_energies=True, use_md_energies=True)
+
+    # Tyk2 -- Run point and MD energy validation tests
+    run_unsampled_endstate_energies('tyk2', use_point_energies=True, use_md_energies=True)
```

Comment:
Just a comment, nothing blocking. Sounds like a good test to parametrize in the future, I'll add that to our [test cleanup issue](https://github.com/choderalab/perses/issues/1013).